tests/run-make/issue-47551/Makefile

```diff
@@ -1,10 +0,0 @@
-# only-linux
-# ignore-32bit
```

Comment:
Suggestion: we can try this original Makefile without ignore-32bit and see if it fails with the same thing.